### PR TITLE
Mitigation against #224

### PR DESCRIPTION
**Pull request by jkburges**
Run the SQL used to generate monthly reports as part of the integration test suite to at least ensure SQL validity.

### CODE REVIEW

**Comment by jkburges** [comment]
@xhoenner tests for monthly report SQL are added in this PR.  Any changes or additional SQL should be put in the directory if it needs to be tested.

**Comment by xhoenner** [comment]
@jkburges sounds good, thanks for letting me know.

**Comment by jkburges** [comment]
Let me know if you are interested in being able to run these tests yourself and/or need help in doing so.

File: src/groovy/au/org/emii/aatams/data/DevelopmentDataInitialiser.groovy [hunk 1]
```diff
@@ -465,7 +465,6 @@ class DevelopmentDataInitialiser extends AbstractDataInitialiser
         WKTReader reader = new WKTReader();
 
         Point location = (Point)reader.read("POINT(30.1234 30.1234)")
-        location.setSRID(4326)
```
Comment (dnahodil) [inline src/groovy/au/org/emii/aatams/data/DevelopmentDataInitialiser.groovy]: I assume this was deliberate? (it seems unrelated)

Comment (jkburges) [inline src/groovy/au/org/emii/aatams/data/DevelopmentDataInitialiser.groovy]: Deliberate - this was causing an error in one of the SQL `GROUP BY` clauses, trying to group geometries with different SRIDs (i.e. 4326 and null)

Comment (dnahodil) [inline src/groovy/au/org/emii/aatams/data/DevelopmentDataInitialiser.groovy]: Ah okay

**Comment by dnahodil** [comment]
This is a good idea. We just need to ensure that we keep all relevant SQL reports in this directory.
